README.md

```diff
@@ -83,7 +83,9 @@ module "dynamodb_table" {
 | `hash_key`                      | ``           | DynamoDB table Hash Key                                                        | Yes      |
 | `range_key`                     | ``           | DynamoDB table Range Key                                                       | Yes      |
 | `ttl_attribute`                 | ``           | DynamoDB table TTL attribute                                                   | No       |
+| `stream_enabled`                | `true`       | Enable DynamoDB streams                                                        | No       |
```

Comment:
default is `false`?

Reply:
oops

Reply:
https://github.com/cloudposse/terraform-aws-dynamodb/pull/9/commits/4bca56c669086bf579b57044a56ffe8dd9f9d497